srcpkgs/roccat-tools/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'roccat-tools'
+pkgname=roccat-tools
+version=5.7.0
+revision=1
+build_style=cmake
+configure_args="-DDEVICES=arvo;isku;iskufx;kiro;kone;koneplus;konepure;konepuremilitary;konepureoptical;konextd;konextdoptical;kovaplus;kova2016;lua;nyth;pyra;ryosmk;ryosmkfx;ryostkl;savu;skeltr;sova;suora;tyon -DWITH_LUA=5.3 -DUDEVDIR=/usr/lib/udev/rules.d -DCMAKE_MODULE_PATH=/usr/share/cmake"
```

Comment:
please cut at 80 columns

Reply:
Fixed, except the first configure param, which alone is nearly 200 characters but cannot be broken.